packages/protocol/engine/notarization/events.go

```diff
@@ -9,17 +9,19 @@ import (
 
 // Events is a container that acts as a dictionary for the events of the notarization manager.
 type Events struct {
-	EpochCommitted *event.Linkable[*EpochCommittedDetails]
-	Error          *event.Linkable[error]
+	EpochCommitted  *event.Linkable[*EpochCommittedDetails]
+	ConflictDropped *event.Linkable[models.BlockID]
```

Comment:
If we keep this we should also add the conflictID

Reply:
We should still support it and log it

Reply:
removed the event along with the ConflictCounter in NotarizationManager